docs/articles/tasks.md

```diff
@@ -0,0 +1,35 @@
+# Tasks
+
+### What is a Task?
+
+Task is a process which is launched by Mesos executor on the Mesos Agent. When Marathon launches an instance of the application definition, Marathon sends a command to Mesos to launch one or more tasks.
+
+### How can I use a Task?
+Tasks are automatically created by Mesos once application definition is being launched.
+
+There are several other places where interaction with Tasks is possible. This includes:
+
+##### API endpoints:
+ * `/v2/apps/<app_id>/tasks`: Lists all running tasks for a provided application id, and gives you task specific information such as agent id, host, task state and so on. It’s also possible to kill tasks using this endpoint.
+ * `/v2/tasks`: Lists all running tasks, and gives you task specific information such as agent id, host, task state and so on.
+ * `/v2/tasks/delete`: Endpoint for deleting one or more tasks
```

Comment:
We tell the user, **what** Marathon have, but we don't say **why** to use this or that. When does the user should fetch `/v2/tasks`? When `/v2/apps/<app_id>/tasks`?  One can delete tasks via both - which one should the user take and why?